test/parallel/test-console-is-a-namespace.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+// https://heycam.github.io/webidl/#es-namespaces
+// https://console.spec.whatwg.org/#console-namespace
+// https://github.com/w3c/web-platform-tests/blob/master/console/console-is-a-namespace.any.js
+
+const common = require('../common');
+const assert = require('assert');
+const { test, assert_equals, assert_true, assert_false } = common.WPT;
+
+assert.doesNotThrow(() => {
+  global.console = global.console;
+});
+
+// Tests above are not from WPT.
+
+/* eslint-disable */
+/* WPT Refs:
+   https://github.com/w3c/web-platform-tests/blob/master/console/console-is-a-namespace.any.js
```

Comment:
Better reference a specific commit hash here (`40e451c` instead of `master`) in case the file changes or turns into a 404